tutorials/pubsub-spring-dedup-messages/index.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Deduplicate Pub/Sub Messages Using Dataflow in a Spring Boot Application
+description: Use Dataflow's PubsubIO to deduplicate Pub/Sub messages in a Spring Boot application.
+author: anguillanneuf
+tags: PubSub, Spring, Spring Cloud GCP, Dataflow, Java
+date_published: 2020-02-29
+---
+
+
+Tianzi Cai | Developer Programs Engineer | Google Cloud
+
+
+## Objectives
+- Configure a Spring Boot application to use [Cloud Pub/Sub] as a message broker.
```

Comment:
Are these brackets for Markdown links?

Reply:
Yes.

Reply:
Will the links populate automatically?

Reply:
Oh yes, all the links are in the bottom of the file. It's to avoid typing the same link for the same phrase over and over again.